differ/differ.go

```diff
@@ -375,18 +377,13 @@ func generateInstantNotificationMessage(clusterURI string, accountID string, clu
 	events := []types.Event{}
 	context := toJSONEscapedString(types.NotificationContext{
 		notificationContextDisplayName: clusterID,
-		notificationContextHostURL:     strings.Replace(clusterURI, "{cluster}", clusterID, 1),
+		notificationContextHostURL:     clusterURI,
 	})
 	if context == "" {
 		log.Error().Msg(contextToEscapedStringError)
 		return
 	}
 
-	//Temporal code to avoid flooding insights-qa owner's email inbox
-	if accountID == "6089719" {
```

Comment:
Now that I have my account directly linked to the account number 6089719, there's no need for this patch